dicom-archive/updateMRI_Upload.pl

```diff
@@ -3,6 +3,90 @@
 # zia.mohades@mcgill.ca
 # Perl tool to update the mri_upload table
 
+=pod
+
+=head1 NAME
+
+updateMRI_Upload.pl - updates database table C<mri_upload> according to an entry in table
+   tarchive
+
+=head1 SYNOPSIS
+
+updateMRI_Upload.pl [options] -profile prod -tarchivePath tarchivePath -source_location source_location
+
+=over 2
+
+=item *
+B<-profile prod> : (mandatory) path (absolute or relative to the current directory) of the 
+    profile file
+
+=item *
+B<-tarchivePath tarchivePath> : (mandatory) absolute path to the tarchive file
+
+=item *
+B<-source_location source_location> : (mandatory) value to set column 
+    C<DecompressedLocation> to for the newly created record in table C<mri_upload> (see below)
+    
+=item *
+B<-globLocation> : Loosen the validity check of the tarchive allowing for the 
+     possibility that the tarchive was moved to a different directory.
+
+=item *
+B<-verbose> : Be verbose
```

Comment:
fix the weird alignment when running the perldoc command

![screen shot 2018-01-17 at 2 59 44 pm](https://user-images.githubusercontent.com/1402456/35064209-49ea7ade-fb97-11e7-91ff-b4c3b948165b.png)

Reply:
Again, I don't see this alignment bug on my end.